RpcServer/RpcServer.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Version>3.0.0-preview1</Version>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <RootNamespace>Neo.Plugins</RootNamespace>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\neo\neo\neo.csproj" />
```

Comment:
We should add references from NuGet.

Reply:
Alright, I will just keep this on the draft to be able to test with neo changes I made. I will make sure to change this before the final PR.

Reply:
Can we use nuget package now?

Reply:
I will need to send my changes on Neo first, but they will break the plugins that rely on RPC, so, it is kinda the egg-or-the-chicken situation hahaha.

I will fix this plugins now, so we can do this whole change faster.

Reply:
You can send the PR to Neo first to let us know what changes are made.